docs/reference/commandline/events.md

```diff
@@ -180,3 +191,22 @@ relative to the current time on the client machine:
     $ docker events --filter 'type=plugin' (experimental)
     2016-07-25T17:30:14.825557616Z plugin pull ec7b87f2ce84330fe076e666f17dfc049d2d7ae0b8190763de94e1f2d105993f (name=tiborvass/no-remove:latest)
     2016-07-25T17:30:14.888127370Z plugin enable ec7b87f2ce84330fe076e666f17dfc049d2d7ae0b8190763de94e1f2d105993f (name=tiborvass/no-remove:latest)
+
+**Format:**
+
+    # docker events --format '{{if eq .Type "container"}}Type={{.Type}}  Status={{.Status}}  ID={{.ID}}{{end}}'
```

Comment:
Wonder if we should use an example that doesn't use "Type" in the template's discussion (using `--filter type=container` is probably more useful to do that?)